docs/_pages/examples.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Examples
+layout: collection
+permalink: /examples/
+collection: examples
+entries_layout: grid
```

Comment:
I prefer the single page with a list of examples for now. It requires less clicking through for users. With the sidebar listing all examples it is also quite easy to navigate.

Eventually I hope to have detailed docs about topics covering multiple examples and keep them with description. For now though, I'd go with single page.